.github/PULL_REQUEST_TEMPLATE/docs.md

```diff
@@ -0,0 +1,37 @@
+## Description
+
+Closes: #XXXX
+
+<!-- Add a description of the changes that this PR introduces and the files that
+are the most critical to review. -->
+
+<!-- Pull requests that sit inactive for longer than 30 days will be closed.  -->
+---
+
+### Author Checklist
+
+*All items are required. Please add a note to the item if the item is not applicable and
+please add links to any relevant follow up issues.*
+
+I have...
+
+- [ ] included the correct `docs:` prefix in the PR title
+- [ ] targeted the correct branch (see [PR Targeting](https://github.com/cosmos/gaia/blob/main/CONTRIBUTING.md#pr-targeting))
+- [ ] provided a link to the relevant issue or specification
+- [ ] reviewed "Files changed" and left comments if necessary
```

Comment:
This point needs more context imo, is it asking to leave comments on one's own PR?

Reply:
Yes, why certain files were changed if the reason is not obvious. For example, usually a deps bump shouldn't result in other files changed besides `go.mod` and `go.sum?